src/context.cpp

```diff
@@ -596,61 +699,172 @@ namespace librealsense
             if ( profiles.size() > 0 )
             {
                 _dev->open( realdds_profiles );
+
             }
 
             software_sensor::open( profiles );
         }
 
+        void handle_video_data( realdds::topics::device::image && dds_frame,
+                                std::shared_ptr< stream_profile_interface > prof, std::string name )
```

Comment:
&

Reply:
Instead of passing `name`, why not pass the syncer to it doesn't have to be looked up?

Reply:
> &
Will add.


> Instead of passing `name`, why not pass the syncer to it doesn't have to be looked up?
Previously `this` was not bound to the calling lambda, but I refactored to remove some temp stack objects and bind it now so I guess we can use the syncer as parameter. Don't think it matters much.

Reply:
Using the syncer as parameter